ChildProject/converters.py

```diff
@@ -403,33 +389,78 @@ def convert(filename: str) -> pd.DataFrame:
+        'Participant': 'CHI', # This is the generic role for adult participants in interviews and other conversations.  Usually, these are coded as having a Participant and an Investigator.  Other forms of this role include Patient, Informant, and Subject which can be listed in the Specific Role field or else just omitted.
+        'Investigator': 'NA', #  Other terms for this role can be listed in the Specific Roles.  These include Researcher, Clinician, Therapist, Observer, Camera_Operator, and so on. 
+        'Partner': 'NA', #  This is the role for the person accompanying the Participant to the interview or conversation.
+        'Boy': 'OCH', #  This is a generic role.
+        'Girl': 'OCH', # This is a generic role.
+        'Adult': 'NA', #  This is a very generic role for use when little else is known.
+        'Teenager': 'NA', # This is a generic role.
+        'Male': 'MAL', # Use this role when all we know is that the participant is an adult male.
+        'Female': 'FEM', #  Use this role when all we know is that the participant is an adult female.
+        'Visitor': 'NA', #  This role assumes that the visitor is coming to a conversation in the home.
+        'Friend': 'OCH', # This is a role for a Friend of the target participants.
+        'Playmate': 'OCH', #  This is a role for a child that the Target_Child plays with.
+        'Caretaker': 'NA', #  This person takes care of the child. Other names for the Specific Role field include Housekeeper, Nursemaid, or Babysitter.
+        'Environment': 'NA', # This role is used in the SBCSAE corpus.
+        'Group': 'NA', # This role is used when transcribing simultaneous productions from a whole group.
+        'Unidentified': 'NA', #  This is a role for unidentifiable participants.
+        'Uncertain': 'NA', # This role can be used when it is not clear who produced an utterance.
+        'Other': 'NA', # This is a generic role.  When it is used, there should be further specification in the Specific Role field. Roles defined by jobs such as Technician, Patron, Policeman, etc can be listed as Other and the details given in the Specific Role field.
+        'Text': 'NA', # This role is used for written segments of TalkBank.
+        'Media': 'NA', #  This role is used for speech from televisions, computers, or talking toys.
+        'PlayRole': 'NA', # This role is used when speakers pretend to be something, such as an animal or another person.
+        'LENA': 'NA', # This role is used in HomeBank LENA recordings.  The specific LENA role is then listed in the Specific Role field.
+        'Justice': 'NA', #  This is role is used in the SCOTUS corpus. It also includes the role of Judge.
+        'Attorney': 'NA', # This is the general role for attorneys, lawyers, prosecutors, etc.
+        'Doctor': 'NA', # This is the general role for doctors.
+        'Nurse': 'FEM', # This is the general role for nurses.
```

Comment:
this should be NA (could be a male nurse)

Reply:
oops that's embarassing!